src/components/network/network-panel.js

```diff
@@ -137,6 +141,14 @@ genotet.NetworkPanel.prototype.setEdgeInfo_ = function(edge, container) {
     .text(edge.target.label);
   container.children('#weight').children('span')
     .text(edge.weight);
+  container.children('#rm-edge').children('button').click(function() {
+    this.signal('update', {
+      type: 'delete-edge',
+      source: edge.source.label,
```

Comment:
You can access the ids of the source and target nodes of an edge by first accessing the edge object, can't you?

If you really need to explicitly give source and target names:
* You need to use their ids instead of their labels.
* You may want to make your deleteEdge() accept two arguments for source and target respectively.